frontend/view-edit-client/client-history/client-history.component.tsx

```diff
@@ -47,7 +47,7 @@ export default function ClientHistory(props: ClientHistoryProps) {
   return (
     <div {...scope} className="card">
       <div className="client-history-header">
-        <h3>Client history</h3>
+        <h1>Client history</h1>
```

Comment:
Same here. The client's name is already an h1, so this may need to be an h2.

Reply:
Axe disagrees with you, so I'm going to stick to what it says :)